src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -231,6 +237,63 @@ def container_logs(cmd, resource_group_name, name, container_name=None, follow=F
             stream_args=(logs_client, resource_group_name, name, container_name, container_group.restart_policy))
 
 
+def container_exec(cmd, resource_group_name, name, container_name=None, exec_command=None, terminal_row_size=None, terminal_col_size=None):
+    """Start exec for a container. """
+
+    if platform.system() is WINDOWS_OS_NAME:
+        print("The container exec operation is currently not supported for Windows.")
+        return
+
+    start_container_client = cf_start_container(cmd.cli_ctx)
+    container_group_client = cf_container_groups(cmd.cli_ctx)
+    container_group = container_group_client.get(resource_group_name, name)
+
+    # Set defaults for rows and cols size
+    if terminal_row_size is None:
+        terminal_row_size = 24
+
+    if terminal_col_size is None:
+        terminal_col_size = 80
+
+    # If container name is not present, use the first container.
+    if container_name is None:
+        container_name = container_group.containers[0].name
```

Comment:
Unless there is a compelling reason that the first container is the one the user would most like to use or the most frequently used, consider simply making this a required parameter.